src/routes/add-custom-discipline/AddCustomDisciplineScreen.tsx

```diff
@@ -30,22 +33,31 @@ const Description = styled.Text`
   padding: 10px 0;
 `
 
-const StyledTextInput = styled.TextInput<{ errorMessage: string }>`
-  font-size: ${props => props.theme.fonts.largeFontSize};
-  font-weight: ${props => props.theme.fonts.lightFontWeight};
-  letter-spacing: 0.11px;
-  font-family: ${props => props.theme.fonts.contentFontRegular};
-  color: ${prop => prop.theme.colors.lunesBlack};
+const InputContainer = styled.View<{ errorMessage: string }>`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
   width: 80%;
   border: 1px solid
     ${props =>
       props.errorMessage ? props.theme.colors.lunesFunctionalIncorrectDark : props.theme.colors.lunesGreyDark};
   border-radius: 4px;
   margin-top: ${hp('8%')}px;
-  padding-left: 15px;
+  padding: 0 15px;
   height: ${hp('8%')}px;
 `
 
+const StyledTextInput = styled.TextInput`
+  flex-grow: 2;
+  font-size: ${props => props.theme.fonts.largeFontSize};
+  font-weight: ${props => props.theme.fonts.lightFontWeight};
+  letter-spacing: 0.11px;
+  font-family: ${props => props.theme.fonts.contentFontRegular};
+  color: ${prop => prop.theme.colors.lunesBlack};
+  width: 80%;
```

Comment:
```suggestion
  padding-right: 15px
```

Reply:
This would not work, as when entering a long code, the icon is moved to the right. Adjusted it nevertheless to width 90% and removed flex.